packages/discord.rb

```diff
@@ -0,0 +1,40 @@
+require 'package'
+
+class Discord < Package
+  description 'Discord is a proprietary, cross-platform, all-in-one voice and text chat application.'
+  homepage 'https://discord.com/'
+  version '0.0.27'
+  license 'Proprietary'
+  compatibility 'x86_64'
+  source_url 'https://dl.discordapp.net/apps/linux/0.0.27/discord-0.0.27.tar.gz'
+  source_sha256 'e9f1da88f05cbfb4d0561f936ad20461767f2f03e69c2994fd059728581447b5'
+
+  # extracted from discord .deb archive
+  depends_on 'alsa_lib'    # libasound2
+  depends_on 'gcc_dev'     # libatomic1, libstdc++6
+  depends_on 'libgconf'    # libgconf-2-4
+  depends_on 'libnotify'   # libnotify4
+  depends_on 'nspr'        # libnspr4
+  depends_on 'nss'         # libnss3
+  depends_on 'libxss'      # libxss1
+  depends_on 'libxtst'     # libxtst6
+  depends_on 'llvm_stage1' # libc++1
+  depends_on 'sommelier'
+
+  no_compile_needed
+
+  def self.install
+    prefix         = "#{CREW_DEST_PREFIX}/share/discord"
+    symlink_prefix = prefix.delete_prefix(CREW_DEST_DIR)
```

Comment:
Why not use a relative path for clarity?

Reply:
Do you mean using relative path as target in symlinks?

Reply:
Something like `./relative/path`.

Reply:
Relative path will not work with different directories, like `../share/discord` will not work in `/usr/local/share/applications`

Reply:
How about an absolute path then?  Something like `#{CREW_PREFIX}/path/to/file`?